pyuvdata/tests/test_ms.py

```diff
@@ -77,6 +64,24 @@ def test_readMSreadUVFITS():
     # them to be the same anyways.
     ms_uv.history = ""
     uvfits_uv.history = ""
+
+    # the objects won't be equal because uvfits adds some optional parameters
+    nt.assert_false(uvfits_uv == ms_uv)
+    # they are equal if only required parameters are checked:
+    nt.assert_true(uvfits_uv.__eq__(ms_uv, check_extra=False))
+
+    # set those parameters to none to check that the rest of the objects match
+    for p in uvfits_uv.extra():
```

Comment:
We could. I'm agnostic about it but I think this is the first time it's come up. The issue here is comparing a CASA exported uvfits to the matching measurement set and I actually suspect that some of these parameters might be available in measurement sets. I have a question out to Aaron about that (although it might have gotten lost in the noise on the channel. I'll email him about it specifically)